easysid.cpp

```diff
@@ -146,28 +189,40 @@ EXPORT void set_filter_enable(uint8_t voice)
 EXPORT void set_filter_lp(bool enable)
 {
     char reg = sid.read_state().sid_register[24];
-    reg |= 0x10;
+    reg &= ~0x10;
+    if (enable) {
```

Comment:
Use an `else` clause here to handle the `reg &= ~0x10` case. Same for the other `enable` functions.